### ISSUE TITLE
A little bit dumb question about loss functions

### ISSUE THREAD

**Issue by lev4ukpavel2** [keras-team/keras#10922]
Hello, I am new to neural networks and I am trying to make a convolutional neural network. 
I don't know which loss function do i need to use.
I take an image as an input, and output is a numpy array [1,1,0,0] or [1,0,0,1], [1,0,1,1] etc.
You can't use categorical crossentropy because you need one to be 1 and others to be 0, right? So, what I should use?
And, what activation i should apply as the last layer of the model?

**Comment by Dref360** [comment]
Hi!
This issue isn't related to a bug/enhancement/feature request or other accepted types of issue.

To ask questions, please see the following resources : 
* https://keras.io/
* https://keras.io/#support
* https://gitter.im/Keras-io/Lobby

Thanks!

If you think I made a mistake, please open another issue.
